from afs.exceptions.AfsError import AfsError

class PtError(AfsError):